docs/zh_cn/understand_mmcv/hook.md

```diff
@@ -0,0 +1,664 @@
+runner.register_hook(ema_config, priority='NORMAL')
+```
+
+##### 恢复训练
+
+当中断训练之后恢复训练，我们需要设置 resume_from 参数（注意，这个参数是一定要设置的，否则和不中断训练的结果是不一致的），这样我们就可以加载 ema 参数，以保证使用 EMAHook 训练的正确性。
+
+```python
+ema_config = dict(type="EMAHook", resume_from="/path/of/your/checkpoint.pth")
+runner = EpochBasedRunner(...)
+runner.register_hook(ema_config, priority='NORMAL')
+```
+
+#### LrUpdaterHook
+
+学习率决定每次更新的步长，合适的学习率可以使训练快速收敛。MMCV 中提供很多学习率衰减策略，其中部分学习率衰减策略也伴有动量衰减策略。
+
+下表是 MMCV 提供的学习率衰减策略和对应的动量衰减策略
+
+| 名称 | 描述 | 策略 | API 文档 |
+| ---  | --- | --- | --- |
+| FixedLrUpdaterHook | 固定学习率 | fixed | [mmcv.runner.FixedLrUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.FixedLrUpdaterHook) |
+| StepLrUpdaterHook, StepMomentumUpdaterHook | 等间隔调整学习率 | step | [mmcv.runner.StepLrUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.StepLrUpdaterHook), [mmcv.runner.StepMomentumUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.FixedLrUpdaterHook) |
+| ExpLrUpdaterHook | 指数调整学习率 | exp | [mmcv.runner.ExpLrUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.ExpLrUpdaterHook) |
+| PolyLrUpdaterHook | 多项式调整学习率 | poly | [mmcv.runner.PolyLrUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.PolyLrUpdaterHook) |
+| InvLrUpdaterHook | 反比例调整学习率 | Inv | [mmcv.runner.InvLrUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.InvLrUpdaterHook) |
+| CosineAnnealingLrUpdaterHook, CosineAnnealingMomentumUpdaterHook | 余弦退火调整学习率 | CosineAnnealing | [mmcv.runner.CosineAnnealingLrUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.CosineAnnealingLrUpdaterHook) |
+| FlatCosineAnnealingLrUpdaterHook | 初始学习率训练一段时间再使用余弦退火调整学习率 | FlatCosineAnnealing | [mmcv.runner.FlatCosineAnnealingLrUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.FlatCosineAnnealingLrUpdaterHook) |
+| CosineRestartLrUpdaterHook | 多次余弦退火调整学习率，每次的初始学习率可能不一样 | CosineRestart | [mmcv.runner.CosineRestartLrUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.CosineRestartLrUpdaterHook) |
+| CyclicLrUpdaterHook, CyclicMomentumUpdaterHook | 循环调整学习率 | cyclic | [mmcv.runner.CyclicLrUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.CyclicLrUpdaterHook), [mmcv.runner.CyclicMomentumUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.CyclicMomentumUpdaterHook) |
+| OneCycleLrUpdaterHook, OneCycleMomentumUpdaterHook | 一个循环内调整学习率，学习率从小增大再减小 | OneCycle | [mmcv.runner.OneCycleLrUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.OneCycleLrUpdaterHook), [mmcv.runner.OneCycleMomentumUpdaterHook](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.OneCycleMomentumUpdaterHook) |
+
+学习率调整策略虽然有很多种，但用法如出一辙，下面只举两个常用的用法。
+
+```{tip}
+为了确保学习率的衰减符合期望，我们可以参考 [可视化学习率脚本](https://mmclassification.readthedocs.io/en/latest/tools/visualization.html#learning-rate-schedule-visualization) 在开始训练前可视化学习率的变化。
```

Comment:
There is a little problem with the rendering of links.

Reply:
done